StorageGrain/StorageGrain.cs

```diff
@@ -0,0 +1,32 @@
+﻿using CommunAxiom.Commons.Client.Contracts.Grains.Storage;
+using Newtonsoft.Json.Linq;
+using Orleans.Runtime;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace StorageGrain
+{
+    // HACK: 
+    // [StorageProvider()]
+    public class StorageGrain : IStorageGrain
```

Comment:
This implementation is incomplete, this needs the same pattern as other grains with business object and storage object.

Reply:
Also don't set the storage provider name on the grain, set it for the grainstate specifically (there's a parameter in PersistentStateAttribute for that).